lib/node.ts

```diff
@@ -0,0 +1,9 @@
+import AlgoliaAnalytics from "./insights";
+import { getFunctionalInterface } from "./_getFunctionalInterface";
+import { getRequesterForNode } from "./utils/getRequesterForNode";
+
+const requestFn = getRequesterForNode();
+const instance = new AlgoliaAnalytics({ requestFn });
+const functionalInterface = getFunctionalInterface(instance);
+
+export default functionalInterface;
```

Comment:
I didn't think about this before but it could later useful to compare node vs browser usage, so we might want to annotate the user agent for that.

I opened an issue for it, we don't have to add this now 😄 https://github.com/algolia/search-insights.js/issues/157